core/Size.js

```diff
@@ -113,21 +114,46 @@ function setVec (vec, x, y, z) {
  * @method
  *
  * @param {String|Number} val The Size mode to resolve.
+ * @param {boolean} quiet Whether to throw an error for invalid mode
  *
  * @return {Number} the resolved size mode from the enumeration.
  */
-function resolveSizeMode (val) {
+function resolveSizeMode (val, quiet) {
```

Comment:
`resolveSizeMode` is called while iterating through the list of input params for the customized function.  If it returns a value, then we know we need to retrieve the value for one of `absolute`, `relative`, or `render`.  If it doesn't return a value, then we use the value from the array as is.